charts/jenkins/templates/cache-pvcs.yaml

```diff
@@ -0,0 +1,29 @@
+{{- /* create list of agents */}}
+{{- $_ := set .Values "agentList" (list .Values.agent) }}
+{{- range $name, $additionalAgent := .Values.additionalAgents }}
+    {{- /* merge .Values.agent into additional agent to ensure it at least has the default values */}}
+    {{- $_ := set $.Values "agentList" (append $.Values.agentList (merge $additionalAgent $.Values.agent)) }}
+{{- end }}
+{{- range $agent := .Values.agentList }}
+{{- if $agent.cache.persistence.enabled }}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim

```

Comment:
We should add the namespace here in case `masterslaveKubernetesNamespace` is set.

Reply:
Fixed.